operations/UAS_user_and_admin_topics/User_Access_Service_UAS.md

```diff
@@ -1,26 +1,27 @@
+[Next Topic: UAS Limitations](UAS_Limitations.md)
 
 ## User Access Service \(UAS\)
 
-The User Access Service \(UAS\) is a containerized service managed by Kubernetes that enables application developers to create and run user applications. UAS runs on a non-compute node \(NCN\) that is acting as a Kubernetes worker node.
+The User Access Service \(UAS\) is a service that manages User Access Instances \(UAIs\) which are containerized services under Kubernetes that provide application developers and users with a lightweight login environment in which to create and run user applications. UAIs run on non-compute nodes \(NCN\), specifically Kubernetes Worker nodes.
 
-Users launch a User Access Instance \(UAI\) using the cray command. Users can also transfer data between the Cray system and external systems using the UAI.
+At a high level, there are two ways to configure UAS with respect to allowing users access to UAIs.  The standard configuration involves the use of [Broker UAIs](Broker_Mode_UAI_Management.md) through which users establish SSH login sessions.  When a login session is established to a Broker UAI the Broker UAI either locates or creates a new UAI on behalf of the user and forwards the user's SSH connection to that UAI.  A [legacy configuration](Legacy_Mode_User-Driven_UAI_Management.md) requires users to create their own UAIs through the `cray` CLI.  Once a UAI is created in this way, the users can use SSH to log into the UAI directly.  The legacy configuration will soon be deprecated.  Sites using it should migrate to the Broker UAI based configuration.
```

Comment:
We're close to deprecating the legacy configuration, but my preference would be to replace "The legacy configuration will soon be deprecated" with something like "The legacy configuration will deprecated in a future release."

Reply:
Agreed.  That is better wording.

Reply:
Fixed.